v1.6.5 breaks the embeding support in django/pongo2 template engine

Hi, v1.6.5 currently breaks the embedding support. Running the application with a tool like `packr` fails with an error like this:
```
2020/12/04 23:15:03 stat /web/templates: no such file or directory
panic: stat /web/templates: no such file or directory

goroutine 1 [running]:
log.Panic(0xc0000c1ba8, 0x1, 0x1)
	/home/asrivastava/golang/go/src/log/log.go:351 +0xae
github.com/flosch/pongo2/v4.MustNewLocalFileSystemLoader(0xc0031353b0, 0xe, 0x1)
	/home/asrivastava/go/pkg/mod/github.com/flosch/pongo2/v4@v4.0.1/template_loader.go:27 +0x8a
github.com/gofiber/template/django.(*Engine).Load(0xc001c60870, 0x0, 0x0)
	/home/asrivastava/go/pkg/mod/github.com/gofiber/template@v1.6.6-0.20201204221418-69646d0e66b0/django/django.go:127 +0x11d
```

--- a/django/django.go
+++ b/django/django.go
@@ -9,7 +9,7 @@ import (
 	"strings"
 	"sync"
 
-	"github.com/flosch/pongo2"
+	"github.com/flosch/pongo2/v4"
 	"github.com/gofiber/fiber/v2"
 	"github.com/gofiber/template/utils"
 )
@@ -118,15 +118,18 @@ func (e *Engine) Load() error {
 
 	e.Templates = make(map[string]*pongo2.Template)
 
-	// New pongo2 defaultset
 	baseDir := e.directory
-	// TODO: work around until pongo2 support native http.FileSystem in stable release
-	if e.fileSystem != nil {
-		baseDir = fmt.Sprint(e.fileSystem)
-	}
-	pongoloader := pongo2.MustNewLocalFileSystemLoader(baseDir)
-	pongoset := pongo2.NewSet("default", pongoloader)
 
+	var pongoloader pongo2.TemplateLoader
+	if e.fileSystem != nil {
+		// ensures creation of httpFileSystemLoader only when filesystem is defined
+		pongoloader = pongo2.MustNewHttpFileSystemLoader(e.fileSystem, baseDir)
+	} else {
+		pongoloader = pongo2.MustNewLocalFileSystemLoader(baseDir)
+	}
+
+	// New pongo2 defaultset
+	pongoset := pongo2.NewSet("default", pongoloader)
 	// Set template settings
 	pongoset.Globals.Update(e.funcmap)
 	pongo2.SetAutoescape(false)

--- a/go.mod
+++ b/go.mod
@@ -1,6 +1,6 @@
 module github.com/gofiber/template
 
-go 1.14
+go 1.15
 
 require (
 	github.com/CloudyKit/jet/v3 v3.0.1
@@ -9,7 +9,7 @@ require (
 	github.com/aymerick/raymond v2.0.2+incompatible
 	github.com/cbroglie/mustache v1.2.0
 	github.com/eknkc/amber v0.0.0-20171010120322-cdade1c07385
-	github.com/flosch/pongo2 v0.0.0-20200913210552-0d938eb266f3
+	github.com/flosch/pongo2/v4 v4.0.1
 	github.com/gofiber/fiber/v2 v2.2.2
 	github.com/mattn/go-slim v0.0.0-20200618151855-bde33eecb5ee
 	github.com/valyala/bytebufferpool v1.0.0

